docs/crds/httprule.md

```diff
@@ -115,6 +115,15 @@ While AKO can terminate TLS traffic, it also provides and option where the users
 In case of reencrypt, if `destinationCA` is specified in the HTTPRule CRD, as shown in the example, a corresponding PKI profile is created for that Pool (host path combination).
 Also Note that only one of `pkiProfile` or `destinationCA` can be provided to configure reencrypt for a Pool corresponding to the host path backend Service.
 
+#### Enable HTTP/2 protocol support for backend
```

Comment:
Are we enabling this setting for `v1beta1` only?

Reply:
Will it be helpful if we specify combination of frontend and backend (HTTP2 enabled and disabled) supported by AKO with this so that customer will get an idea how to enable end to end HTTP2 setting?

Reply:
@akshayhavile I have added the combination as suggested by you.